### ISSUE TITLE
[BUG] - Enter a descriptive title

### ISSUE THREAD

**Issue by 997862** [vultr/whmcs-vultr#69]
1. Go to 'After purchasing and opening VPS service through WHMCS, fill in hostname, password and NS1 / 2, the order will arrive as soon as the payment is completed (Figure 01), continue to fill in hostname information and create.'
2. Click on 'Until receiving the email from vultr, VPS has been opened, but the service has been purchased by WHMCS, which is still not opened after purchase. You can continue to open VPS after filling in hostname.(Figure 02 03)'
3. Scroll down to 'According to https://www.vultr.com/api/ test, it is proved that API is OK. My server has been migrated to Hong Kong, China, and the speed of arriving at vultr is very stable. (Figure 04）'

Operating system: Ubuntu server 18.04
Server physical address: Hong Kong, China
WHMCS version: 7.9.1
PHP version: 7.3.14

<img width="1919" alt="01" src="https://user-images.githubusercontent.com/1093092/73520189-03c31880-443e-11ea-8a61-ae6c0d3cc297.png">

<img width="1334" alt="02" src="https://user-images.githubusercontent.com/1093092/73520208-10477100-443e-11ea-93cf-d11469a5802b.png">

<img width="1260" alt="003" src="https://user-images.githubusercontent.com/1093092/73520230-1dfcf680-443e-11ea-9760-90178ebb70bc.png">

<img width="791" alt="04" src="https://user-images.githubusercontent.com/1093092/73520250-281ef500-443e-11ea-9d89-b4cbcb661c8f.png">

**Comment by 997862** [comment]
Who will tell me how to solve it?

**Comment by breakforme732** [comment]
@997862 I'm not seeing an issue in your screenshots. It looks like you used whmcs to create two instances and two instances were created. There are, however, two issues with your last screenshot
1. The error you are receiving from the API is because you are using an invalid SUBID in your curl
2. You posted your private API key to the internet, which is extremely unsafe. I would suggest removing that screenshot.

**Comment by 997862** [comment]
Yes, the creation is completed. When the creation is completed, it should be completed with orders. It can be managed in WHMCS, but now it cannot be managed. It has been shown that it has not been created. After multiple submissions, multiple VPS services will be opened.

**Comment by 997862** [comment]
Please help me.

**Comment by eran101** [comment]
> Yes, the creation is completed. When the creation is completed, it should be completed with orders. It can be managed in WHMCS, but now it cannot be managed. It has been shown that it has not been created. After multiple submissions, multiple VPS services will be opened.

This may be the issue:
https://github.com/vultr/whmcs-vultr/issues/70#issuecomment-593870840

**Comment by 997862** [comment]
Through the account login view, VPS has been created for normal use, I want to know how to solve this problem?
